internal/config/migrate/from_1_17.go

```diff
@@ -60,11 +60,10 @@ func migrateFrom1_17(cfg *config.Config) error {
 	}
 
 	// Upgrade pause image
-	// https://github.com/cri-o/cri-o/pull/3582
-	logrus.Infof("Checking for pause_image, which now should be k8s.gcr.io/pause:3.2 instead of k8s.gcr.io/pause:3.1")
-	newPauseImage := "k8s.gcr.io/pause:3.2"
+	// https://github.com/cri-o/cri-o/pull/4550
+	logrus.Infof("Checking for pause_image, which now should be %s instead of k8s.gcr.io/pause:3.1", config.DefaultPauseImage)
 	if cfg.PauseImage == "k8s.gcr.io/pause:3.1" {
-		cfg.PauseImage = newPauseImage
+		cfg.PauseImage = config.DefaultPauseImage
 		logrus.Infof(`Changing "pause_image" to %s`, cfg.PauseImage)
```

Comment:
Does this imply there should be a a migrator from 1.18, 1.19, and 1.20 too? It seems like someone migrating from 1.17 (or earlier...) would end up with the new pause image, but migrating from 1.18, 1.19, and 1.20 to cri-o 1.21 would still have the older pause image in their config.

That said, poking around, #3929 also changed a default config, but didn't implement a migrator... So perhaps this isn't a thing anymore? cri-o 1.17 is out-of-maintenance now.

Reply:
- 1.17 is using k8s.gcr.io/pause:3.1
- 1.18~1.20 is using k8s.gcr.io/pause:3.2 
If so, I need to add some logic for it. 

Sorry for some misunderstanding here as this is my first PR in cri-o.